Docker_cram/dockerfile

```diff
@@ -2,103 +2,60 @@
 FROM osrf/ros:noetic-desktop-full
 SHELL ["/bin/bash", "-c"]
 
-#source ROS
-RUN source /opt/ros/noetic/setup.bash
-
-#Debug
-RUN echo $CMAKE_PREFIX_PATH
-RUN echo $ROS_PACKAGE_PATH
+ENV ROSINSTALL_PREFIX=https://raw.githubusercontent.com/artnie/cram/jupyter-noetic-devel
```

Comment:
It looks structured. My suggestion:- instead of pulling from your repo let's use cram/cram raw file as Alina (n other editors) always do updates there and hence it will everytime update itself automatically

Reply:
the problem with the official cram repository is, that it doesn;t have the `jupyter` directory with the common-lisp-kernel installer script yet. and also, the cram devel branch sometimes breaks and does stupid stuff, so i want to mitigate the damage for the jupyter notebook as much as possible, to have a working cram version there. if the cram version we use for docker is malfunctioning, i will explicitly update the cram repo that we use. eventually i want to merge my fork and branch with the original cram repo, but not before the fall school, that's too risky.